### ISSUE TITLE
Support custom BASE_PATH 

### ISSUE THREAD

**Issue by anhao** [3ru/gpt-translate#25]
default BASE_PATH is : https://api.openai.com/v1

Can you support custom BasePATH?

**Comment by 3ru** [comment]
@anhao 

I am unfamiliar with `BASE_PATH`, is it something like [this](https://github.com/openai/openai-node/blob/v4/examples/azure.ts#L21)?

I would like to [wait for the official release](https://github.com/openai/openai-node/issues/139#issuecomment-1627903217) of `openai-node` [version 4](https://github.com/openai/openai-node/discussions/182), but I will include the beta version if it is strongly needed for your purposes.

**Comment by anhao** [comment]
@3ru 

If BasePath is set in the configuration, then the set BasePath will be used, not https://api.openai.com/v1

https://github.com/openai/openai-node/blob/dc821be3018c832650e21285bade265099f99efb/base.ts#L56

**Comment by anhao** [comment]
hahha , Can you use gpt-translate to make a Chinese document?  https://g-t.vercel.app/

**Comment by anhao** [comment]
```
/gpt-translate *.md *.cn.md Chinese
```

**Comment by 3ru** [comment]
> hahha , Can you use gpt-translate to make a Chinese document? [g-t.vercel.app](https://g-t.vercel.app/)

Yes, absolutely.
However, the documentation in this repository uses the `mdx` extension, which is not yet supported, so I will work on that soon.

**Comment by 3ru** [comment]
@anhao 
My mistake, apparently the `openai` library already supported changing the basePath, so I made it possible to change it in `v1.1.4-alpha`.

Please try updating the following parts of the workflow file.
1. `uses`
2. `basePath`
```yml
uses: 3ru/gpt-translate@v1.1.4-alpha
basePath: ''
```

**Comment by 3ru** [comment]
This problem is probably solved in `v1.1.4-alpha`.
If you have any problems, feel free to re-open this.
